src/Discord.Net/DiscordClientConfig.cs

```diff
@@ -0,0 +1,13 @@
+namespace Discord
+{
+    public class DiscordClientConfig
+    {
+        public string? Token { get; set; }
```

Comment:
Does this mean that TokenType is dead, and will be assumed based on the type of client that is used?